docs/source/open-edx-background.rst

```diff
@@ -0,0 +1,39 @@
+Open edX Background
+===================
+
+What is Open edX?
+--------------------
+Open edX refers to its two components: the Open edX Platform and the Open edX Community.
+The Open edX Platform, a scaleable web-based platform that empowers both course authors and learners, consists of the Learning Management System (LMS) and the Content Management System (CMS).
```

Comment:
I think it'd be good to mention "Studio" here as well, as an alternate name we use for the CMS.

Reply:
Added